plugins/ufm_rust_client/README.md

```diff
@@ -0,0 +1,71 @@
+## *UFM rust client*
+The rust client of UFM
+
+### *Build*
+To install [rust](https://www.rust-lang.org/tools/install) on the machine.
+
+```
+cargo build
+```
+
+### *Usage*
+
+Use the following syntax to run `ufmcli` commands from your terminal window:
+```
+ufmcli [flags] [command] [parameters]
+```
+
+where flags, command, and parameters are:
+* `flags`: Specifies optional flags. For example, you can use the `--ufm-address` or `--ufm-token` flags to specify the address and token of the UFM server.
+* `command`: Specifies the operation that you want to perform on UFM, for example `view`, `list`, `delete`, `create`.
+* `parameters`: Specifies the parameters for command, for example `-p` or `--pkey` to specify the pkey for the partition.
+
+`ufmcli` needs to know the UFM server address and authentication(username/password or token) to access UFM server. 
+
+There are two ways to set the UFM server address and authentication for `ufmcli` command.
+* Environment: Set environment `UFM_ADDRESS` for the UFM server address, for example `export UFM_ADDRESS=http://hpc-cloud-gw.mtr.labs.mlnx:4402`. Set environment `UFM_TOKEN` or `UFM_USERNAME`, `UFM_PASSWORD` for the authentication.
+* Flag: Specifies the flags in the `ufmcli` command, you can use `--ufm-address` to specify the UFM address and `--ufm-token` or `--ufm-username`, `--ufm-password` to specify the authentication.
+
+
+#### *Version*
+```
+./ufmcli version
+6.11.1-2
```

Comment:
Why do we need hardcoded UFM version here?
can we support regex or $UFM_VERSION > 6.11.1-2 != 6.12.1-5 e.t.c.?